### PR TITLE
feat(client): Send a `signup_must_verify` message to the firstrun page on signup.

### LINKED ISSUE #2790: firstrun - send a signal to the parent on successful signup
This is so the Growth team can track when users have successfully signed up.

https://bugzilla.mozilla.org/show_bug.cgi?id=1150231#c20

ref #2101, #2791

### PR DESCRIPTION
**Pull request by shane-tomlinson**
Triggered whenever a user successfully completes the signup form, or an
unverified user successfully completes the signin form.

fixes #2790
ref #2101 

@jpetto - this is for you!

### CODE REVIEW

**Comment by shane-tomlinson** [comment]
@jpetto - this is running on https://stomlinson.dev.lcip.org, `verify_signup` is the new message.

**Comment by jpetto** [comment]
I'm seeing the new message. Thanks! :+1:

**Comment by shane-tomlinson** [comment]
Thanks @jpetto!

File: app/scripts/models/auth_brokers/first-run.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ define([
     _iframeCommands: {
        LOADED: 'loaded',
        LOGIN: 'login',
+       VERIFY_SIGNUP: 'verify_signup',
```
Comment (shane-tomlinson) [inline app/scripts/models/auth_brokers/first-run.js]: Is `verify_signup` a reasonably descriptive message, or would `must_verify_signup` be better?

**Comment by philbooth** [comment]
WFM LGTM :+1: :shipit:

File: app/scripts/models/auth_brokers/first-run.js (hunk 1 above)
Comment (shane-tomlinson) [inline app/scripts/models/auth_brokers/first-run.js]: From @jpetto in IRC:
> [15:16:30]  <jpetto>	ah - wasn't sure if that question was for me or not ;)
[15:16:56]  <jpetto>	it doesn't quite fit with the other message names
[15:17:16]  <jpetto>	the others describe what happened - resize, verification_complete, login
[15:17:33]  <jpetto>	so yeah, i'd say must_verify_signup is better
[15:18:10]  <jpetto>	or even signup_unconfirmed
[15:18:18]  <jpetto>	or signup_unverified

**Pull request merged**